asset-transfer-basic/application-java/src/test/java/application/java/TestApp.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright IBM Corp. All Rights Reserved.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+// Running tests: 
+// gradle test --tests TestApp.start
+// gradle test --tests TestApp.CreateAssetTest
+// gradle test --tests TestApp.ReadAssetTest
+// gradle test --tests TestApp.DeleteAssetTest
+// gradle test --tests TestApp.UpdateAssetTest
+// gradle test --tests TestApp.TransferAssetTest
+// gradle test --tests TestApp.GetAllAssetsTest
+
+package application.java;
+
+import org.junit.Test;
+import static org.junit.Assert.*;
+
+
+public class TestApp {
```

Comment:
I think a main method is better to demo an app, than a unit test, since each test is disconnected.
Not sure if we decided on one way or other, during the brief time in the call this morning.

My suggestion is to have a main() method (not in the src/test), that calls a sequence of invoke & query with some Println like in js app.

and, Use  connect() only once in a sample transaction sequence.

Reply:
I restructured it in this exact way, a main file will be used instead to showcase the app.